packages/nimble-components/src/tabs/styles.ts

```diff
@@ -22,5 +22,6 @@ export const styles = css`
         grid-row: 2;
         grid-column-start: 1;
         grid-column-end: 4;
+        overflow: auto;
```

Comment:
It'd be nice if we could find a simple way to autotest this.

A chromatic test might work if browsers visibly render scrollbars and those get included in the snapshot, but I'm not sure if that's the case. 

Alternatively, a unit test that [uses `scrollHeight`](https://stackoverflow.com/questions/4814398/how-can-i-check-if-a-scrollbar-is-visible) might work.

Reply:
Changed the matrix tests to have long text that triggers both vertical and horizontal scrollbars.

Reply:
Sorry to reopen this, but if we're going the matrix test route then I'd vote to still include a test with short content so that we can verify that scrollbars don't appear in that case. I hope that should be easy to do by making the content be another parameter in the matrix state.

Reply:
Milan suggested making it a separate story, which I liked. So the matrix story is now back the way it was.